_data/projects.yml

```diff
@@ -58,34 +58,35 @@ projects:
       image: "img/projects/tuf_diagram.png"
       site: "https://theupdateframework.github.io"
       status: *adopted
-      description: "Software must be updated frequently to not only ensure
-improved operation, but also to patch security flaws.
-Unfortunately, the act of performing an update can itself be a
-way to attack a computer, if a malicious update is installed. TUF is a
-comprehensive, flexible framework to secure software updates even in situations
-where the software repository is compromised. Developers can integrate TUF
-into any software update system, or native programming language due to its
-concise, self-contained architecture and specification. It offers both developers
-and users protection from a host of potential attacks."
+      description: "Software repositories can receive updates on software or
+project metadata as frequently as every few minutes. This growing flow of updates
```

Comment:
I don't like that this initially makes the problem seem like one of volume not security

Reply:
Check revised copy.